evalml/automl/automl_search.py

```diff
@@ -631,6 +635,28 @@ def __init__(
         if not engine:
             self._engine = SequentialEngine()
         else:
+            valid_engines = [
```

Comment:
I feel like we should add `sequential` as a valid engine string as well for completeness

Reply:
Done!